cfpb/consumerfinance.gov
# Paying for College javascript reorganization

This is a small but sensible reorganization of the Paying for College code in `/cfgov/unprocessed/apps/paying-for-college/js/`

When the "Dynamic Disclosures" code was folded into this repo, it was placed in a `disclosures/` folder inside `/cfgov/unprocessed/apps/paying-for-college/js/`. This made the folder a little messy, since the Financial Path to Graduation code's subfolders were in the same folder.

This PR moves those subfolders into a `college-costs/` subfolder. The `college-costs.js` file remains where it is, so no changes are necessary outside of some changes in that file.

## Changes
- Move `utils/`, `dispatchers/`, `views/`, and `models/` into the `college-costs/` folder, all inside `/cfgov/unprocessed/apps/paying-for-college/js/`.

## Notes
**Note:** I had also started changing references to `college-costs` to `path-to-graduation` so the variable naming and code would match the branding of the final site. **However**, this proved to be an immense, tangled undertaking, so I gave up. After all, there probably won't be another Paying for College project in the future which would result in confusion, and anyone looking at the code of the Financial Path to Graduation will clearly see all then references to `college-costs`. But if anyone thinks this is really worth doing, let me know and I'll do it! 😆 

## How to test this PR
1. Just pull it in and make sure everything still works. 😄  (Functional tests are passing, so I don't have major concerns.)

## Checklist
